app/uk/gov/hmrc/nisp/controllers/NIRecordController.scala

```diff
@@ -65,8 +66,9 @@ trait NIRecordController extends NispFrontendController with AuthorisedForNisp w
               niSummary.spaYear))
 
               val tableStart = niSummary.recordEnd.getOrElse(niSummary.earningsIncludedUpTo.taxYear + 1)
+              val currentDate = new LocalDate()
```

Comment:
Ok done. Although you will not be able to test this locally just because it gets the current time for London and not the local machine. 
Tests pass though.